docs/intro/installation.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Installation
+sidebarSortOrder: 1
+---
+
+This section covers the steps to set up your local environment for Solana
+development.
+
+<Steps>
+
+### Install Dependencies
+
+<!-- prettier-ignore -->
+<Tabs groupId="language" items={['MacOS', 'Linux', 'Windows']}>
+<Tab value="MacOS">
+
+On macOS, install the
+[Xcode command line](https://developer.apple.com/xcode/resources/) tools, which
+you can download it directly from Apple.
+
+You can check if the Xcode CLI is installed via this command:
+
+```shell
+xcode-select -p
+```
+
+If you don't see a path returned, you need to install the CLI tools.
+
+#### There are three ways to install Xcode CLI tools:
+
+1. Installing via your terminal using the following command:
+
+```shell
+xcode-select --install
+```
+
+2. Download the installer and install it with a graphical interface
+   [Apple Developer Tools](https://developer.apple.com/download/all/)
+
+   ![Xcode CLI from Apple](/assets/guides/setup-local-environment/setup-xcode.png)
+
+3. Installing via homebrew: we have the following guide to
```

Comment:
can you adjust this to read "we recommend the following guide" since we are sending people to another site and not on solana-com

Reply:
Going to remove the xcode portion, since its not required for Solana development.
Not sure why its got added in the first place since xcode is only required if you're building macOS apps.

Reply:
Agree with you John. Xcode isn't important to us other than potentially adding it as a dependency. Plenty of resources for a dev to go find by themselves.